src/v/coproc/tests/utils/helpers.cc

```diff
@@ -51,3 +53,16 @@ deregister_coprocessors(
     return client.disable_copros(
       std::move(req), rpc::client_opts(rpc::no_timeout));
 }
+
+model::record_batch_reader single_record_record_batch_reader() {
```

Comment:
can you replace this by generating the batch directly, or passing a count value to the existing random record batch reader? what if i wanted 2 batches instead of 1...?

Reply:
I'd need to rebase against my other branch for that, but still that generator didn't allow you to configure the exact number of records per batch. I could change that though

Reply:
up to you